LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2016   
+Crown Copyright (c) 2016-2017
```

Comment:
Chris,

No problem.

Annoyingly UK Law refers to it as Crown Copyright http://www.legislation.gov.uk/ukpga/1988/48/section/163

We could change to "Copyright (c) 2016-2016 Crown Copyright" to be really picky.

Steve